modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/PluginConstants.java

```diff
@@ -205,4 +260,36 @@ private StatusCodes() {
         public static final int MULTI_STATUS_HTTP_CODE = 207;
     }
 
+    /**
+     * Tenant Configuration related constants.
+     */
+    public final class TenantConfigProperties {
+        private TenantConfigProperties() {
+            throw new AssertionError();
+        }
+
+        public static final String LICENSE_KEY = "windowsLicense";
+        public static final String LANGUAGE_US = "en_US";
+        public static final String CONTENT_TYPE_TEXT = "text";
+        public static final String NOTIFIERFREQUENCY = "notifierFrequency";
+        public static final String KEYSTOREPASSWORD = "keystorePassword";
```

Comment:
Same that Harshan has suggested needs to be used for the following variables as well.